syntaxes/prql.tmLanguage.json

```diff
@@ -82,20 +138,55 @@
         {
           "name": "keyword.operator.new",
           "match": "\\{[^}]*}"
+        },
+        {
+          "include": "#escape"
         }
       ]
     },
-    "strings": {
+    "unicode-bidi": {
+      "name": "invalid.illegal",
+      "match": "(\u202A|\u202B|\u202D|\u202E|\u2066|\u2067|\u2068|\u202C|\u2069)"
+    },
+    "string-quoted-raw-single": {
+      "name": "string.quoted.single",
+      "begin": "r'",
+      "end": "'"
+    },
+    "string-quoted-raw-double": {
+      "name": "string.quoted.double",
+      "begin": "r\"",
+      "end": "\""
+    },
```

Comment:
Do these resolve to the same "type" (i.e. are they the same color?). Ideally they should be...

Reply:
Yes, they are the same colors but they're semantically different tokens. I suppose it would be possible for some theme to apply different colors for them though.

Reply:
These grammar files should be labeling a language's text with types, where things that are the same are labeled as the same type.

So it shouldn't be possible for a theme to apply different colors for single & double quoted strings, because they mean the same thing (at the moment...).

Do you agree? Or I'm missing something? 

If so — can we make the same things the same? So we only have one basic string type, for example.

Reply:
Hmm, yeah that makes sense when you say it I suppose. I changed them to just `string.quoted`.

Reply:
Super, thanks!